keyboards/handwired/wulkan/rules.mk

```diff
@@ -0,0 +1,22 @@
+MCU = STM32F303
+
+# Build Options
+#   comment out to disable the options.
+#
+BACKLIGHT_ENABLE = no
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+LAYOUTS = ortho_4x12
```

Comment:
**This is a general comment; you don't need to change anything.**

The reason for the Travis CI errors, is because when you use the [Community Layouts](https://docs.qmk.fm/#/feature_layouts) feature, Travis attempts to build all the keymaps for that layout for your keyboard. The keymaps that are failing are all attempting to compile with RGB Underglow enabled, but your board doesn't support it by default, so the job fails.

Our standard for new keyboards is that all the default and reference keymaps that are shipped with the keyboard addition should compile successfully. You only have one keymap, `default`, in this PR, and it compiles without errors, so you're fine here.

Reply:
Okey, that makes sence. So if I for example would move `RGBLIGHT_ENABLE = no` to the rules for the keymap instead of the keyboard it should also be able to compile those that is failing at the moment?

Reply:
RGB Underglow doesn't currently work on ARM-based microcontrollers in the master branch. A lot of QMKs features are hard-coded to AVR architecture because (ATmega32u4 specifically, IIRC) because it's what was nearly every keyboard PCB used. Support is currently being worked on.

It's also not your responsibility to make sure affected Community Layout keymaps compile, but is the responsibility of the keymap's user.

We also don't like to have a `keymaps/default/rules.mk` file that changes settings that are set at the keyboard level, because it interferes with the API that QMK Configurator uses to compile keymaps, leading to situations where a feature or keyboard works differently depending on how the user got their firmware. We try to avoid this as best as we can.

Reply:
Yeah it would feel wrong to have it in the keymaps! Thanks for the excellent explanation on this topic :)